lib/jquery-rjs/rendering.rb

```diff
@@ -1,13 +1,21 @@
 require 'action_view/helpers/rendering_helper'
 
-ActionView::Helpers::RenderingHelper.module_eval do
-  def render_with_update(options = {}, locals = {}, &block)
-    if options == :update
-      update_page(&block)
-    else
-      render_without_update(options, locals, &block)
-    end
+module JqueryRjs::RenderingHelper
+  method_name = "render#{'_with_update' if RUBY_VERSION < '2'}"
+  define_method(method_name) do |options = {}, locals = {}, &block|
+    update_page(&block) if options == :update
+
+    args = options, locals, block
+    RUBY_VERSION < '2' ? render_without_update(*args) : super(*args)
```

Comment:
Basically.  Again, arguably pedantic, but this is technically changing the existing functionality, when the whole point of this PR as I see it is to retain the same functionality sans deprecation warnings.

What I am suggesting is "on load, if ruby 2, define this method, otherwise this method", instead of "when this method is called, are we still ruby 2?  okay, then execute the same block of code again".  With the second one, unless you can pull off swapping out your ruby runtime mid process (which would be incredibly impressive, and also, why?), then there is no point in that check existing in the method, and it should just determine how it is defined.

That said, saying this is a "performance concern" is probably laughable, but it seems silly to include an in a method when it is an code load evaluation concern, not a method runtime concern.  Silly, but at the same time, not worth investing if there isn't a easy solution (mine poor solution was written not knowing how `alias_method_chain` functions in the first place...)